versions.tf

```diff
@@ -3,9 +3,9 @@ terraform {
     null = {
       source = "hashicorp/null"
     }
-    metal = {
-      source = "equinix/metal"
-      version = "2.1.0" 
+    equinix = {
+      source = "equinix/equinix"
+      version = "1.14.1"
```

Comment:
I think we should leave our [version constraints more open](https://developer.hashicorp.com/terraform/language/expressions/version-constraints#version-constraint-syntax); for example, `~> 1.14`, which would make it easier for users to keep up with the latest non-breaking changes.

Reply:
done